<!doctype html>
<!--[if lt IE 7]><html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en"> <![endif]-->
<!--[if (IE 7)&!(IEMobile)]><html class="no-js lt-ie9 lt-ie8" lang="en"><![endif]-->
<!--[if (IE 8)&!(IEMobile)]><html class="no-js lt-ie9" lang="en"><![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"><!--<![endif]-->
<head>
<meta charset="utf-8">
<title>Research &#8211; phyletica</title>
<meta name="description" content="Some research details.">
<meta name="keywords" content="">


<!-- Twitter Cards -->
<meta name="twitter:title" content="Research">

<meta name="twitter:description" content="Some research details.">

<meta name="twitter:site" content="@jamoaks">
<meta name="twitter:creator" content="@jamoaks">

<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="/images/default-thumb.png">

<!-- Open Graph -->
<meta property="og:locale" content="en_US">
<meta property="og:type" content="article">
<meta property="og:title" content="Research">

<meta property="og:description" content="Some research details.">

<meta property="og:url" content="http://phyletica.org/research/old-index/">
<meta property="og:site_name" content="phyletica">

<meta property="og:image" content="http://phyletica.org/images/default-thumb.png">

<!-- Schema.org markup for Google+ -->
<meta itemprop="name" content="Research">

<meta itemprop="description" content="Some research details.">


<meta itemprop="image" content="http://phyletica.org/images/default-thumb.png">



<!-- Webmaster Tools verfication -->
<meta name="google-site-verification" content="7RL6oMJptXKQ9M8JGuy2wFRfXlScLOKDmiSAQpTOpPQ">



<link rel="canonical" href="http://phyletica.org/research/old-index/">
<link href="http://phyletica.org/feed.xml" type="application/atom+xml" rel="alternate" title="phyletica Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">

<meta http-equiv="cleartype" content="on">

<!-- HTML5 Shiv and Media Query Support -->
<!--[if lt IE 9]>
	<script src="/assets/js/vendor/html5shiv.min.js"></script>
	<script src="/assets/js/vendor/respond.min.js"></script>
<![endif]-->

<!-- Modernizr -->
<script src="/assets/js/vendor/modernizr-2.7.1.custom.min.js"></script>

<link href='//fonts.googleapis.com/css?family=Raleway|Lato|Roboto+Slab:300,400|Roboto:500|PT+Sans+Narrow:400,700%7CPT+Serif:400,700,400italic' rel='stylesheet' type='text/css'>

<!-- Icons -->
<!-- 16x16 -->
<link rel="shortcut icon" href="/favicon.ico">
<!-- 32x32 -->
<link rel="shortcut icon" href="/favicon.png">
<!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
<link rel="apple-touch-icon-precomposed" href="/images/apple-touch-icon-precomposed.png">
<!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
<link rel="apple-touch-icon-precomposed" sizes="72x72" href="/images/apple-touch-icon-72x72-precomposed.png">
<!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
<link rel="apple-touch-icon-precomposed" sizes="114x114" href="/images/apple-touch-icon-114x114-precomposed.png">
<!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
<link rel="apple-touch-icon-precomposed" sizes="144x144" href="/images/apple-touch-icon-144x144-precomposed.png">

<!-- Hack font -->
<link rel="stylesheet" href="//cdn.jsdelivr.net/font-hack/2.015/css/hack.min.css">

</head>

<body class="page">

<!--[if lt IE 9]><div class="browser-upgrade alert alert-info">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div><![endif]-->

<div class="navigation-wrapper">
	<div class="site-name">
		<a href="/">phyletica</a>
	</div><!-- /.site-name -->
	<div class="top-navigation">
		<nav id="site-nav" class="nav">
		    <ul>
		        
				    
                    <li><a href="/" >Home</a></li>
				
				    
                    <li><a href="/people/" >People</a></li>
				
				    
                    <li><a href="/research/" >Research</a></li>
				
				    
                    <li><a href="/teaching/" >Teaching</a></li>
				
				    
                    <li><a href="/software/" >Software</a></li>
				
				    
                    <li><a href="/publications/" >Publications</a></li>
				
				    
                    <li><a href="/posts/" >Posts</a></li>
				
		    </ul>
		</nav>
	</div><!-- /.top-navigation -->
</div><!-- /.navigation-wrapper -->



  <div class="image-wrap">
  <img src=
    
      "/images/header-misty-mangroves.jpg"
    
  alt="Research feature image">
  
    <span class="image-credit">Photo Credit: <a href="http://phyletica.github.io">Jamie Oaks</a></span>
  
  </div><!-- /.image-wrap -->


<div id="main" role="main">
  <div class="article-author-side">
    

<div itemscope itemtype="http://schema.org/Person">


	<img src="/images/phyletica-avatar.png" class="bio-photo" alt="Phyletica bio photo">


  <h3 itemprop="name">Phyletica</h3>
  <p>Exploring biodiversification.</p>
  <a href="mailto:phyletica@auburn.edu" class="author-social" target="_blank"><i class="fa fa-fw fa-envelope-square"></i> Email</a>
  <a href="http://twitter.com/jamoaks" class="author-social" target="_blank"><i class="fa fa-fw fa-twitter-square"></i> Twitter</a>
  
  
  
  
  
  <a href="http://github.com/phyletica" class="author-social" target="_blank"><i class="fa fa-fw fa-github"></i> Github</a>
  
  
  
  
  
  
  <a href="https://youtube.com/channel/UCPPWy11zYqq-9lE_CQ0qPrw" class="author-social" target="_blank"><i class="fa fa-fw fa-youtube-square"></i> Youtube</a>
  
  
  
  <a href="http://phyletica.github.io" class="author-social" target="_blank"><i class="fa fa-fw fa-globe"></i> Interweb</a>
  <a href="http://www.auburn.edu" class="author-social" target="_blank"><i class="fa fa-fw fa-paw"></i> Auburn</a>
  <a href="http://www.auburn.edu/biology" class="author-social" target="_blank"><i class="fa fa-fw fa-university"></i> AU Biology</a>
  <a href="http://www.aumnh.org" class="author-social" target="_blank"><i class="fa fa-fw fa-leaf"></i> AU Museum</a>
  
  
</div>

  </div>
  <article class="page">
    <h1>Research</h1>
    <div class="article-wrap">
      <section id="table-of-contents" class="toc">
  <header>
    <h3><i class="fa fa-book"></i> Contents</h3>
  </header>
<div id="drawer">
<ul id="markdown-toc">
  <li><a href="#overview" id="markdown-toc-overview">Overview</a></li>
  <li><a href="#inferring-shared-evolutionary-history" id="markdown-toc-inferring-shared-evolutionary-history">Inferring shared evolutionary history</a>    <ul>
      <li><a href="#approximate-likelihood-bayesian-inference-of-co-diversification" id="markdown-toc-approximate-likelihood-bayesian-inference-of-co-diversification">Approximate-likelihood Bayesian inference of co-diversification</a></li>
      <li><a href="#implementing-a-full-likelihood-bayesian-model-of-co-diversification" id="markdown-toc-implementing-a-full-likelihood-bayesian-model-of-co-diversification">Implementing a full-likelihood Bayesian model of co-diversification</a></li>
    </ul>
  </li>
  <li><a href="#diversification-across-southeast-asia" id="markdown-toc-diversification-across-southeast-asia">Diversification across Southeast Asia</a>    <ul>
      <li><a href="#diversification-of-geckos" id="markdown-toc-diversification-of-geckos">Diversification of geckos</a></li>
      <li><a href="#comparative-phylogeography-of-the-philippines" id="markdown-toc-comparative-phylogeography-of-the-philippines">Comparative phylogeography of the Philippines</a></li>
      <li><a href="#diversification-across-mainland-southeast-asia" id="markdown-toc-diversification-across-mainland-southeast-asia">Diversification across mainland Southeast Asia</a></li>
    </ul>
  </li>
  <li><a href="#diversification-of-the-highly-endemic-fauna-of-the-gobi-desert" id="markdown-toc-diversification-of-the-highly-endemic-fauna-of-the-gobi-desert">Diversification of the highly endemic fauna of the Gobi Desert</a></li>
  <li><a href="#diversification-in-west-central-african-rainforests" id="markdown-toc-diversification-in-west-central-african-rainforests">Diversification in West-Central African rainforests</a></li>
  <li><a href="#diversification-of-crocodiles" id="markdown-toc-diversification-of-crocodiles">Diversification of Crocodiles</a></li>
  <li><a href="#data" id="markdown-toc-data">Data</a></li>
  <li><a href="#references" id="markdown-toc-references">References</a></li>
</ul>

  </div>
</section>
<!-- /#table-of-contents -->

<h2 id="overview">Overview</h2>

<p>What we do in the phyletica lab: Develop phylogenetic methods, collect genetic
datasets from natural populations, and apply the former to the latter, all with
the goal of ending up with a better understanding of processes of
diversification.
This entails a variety of day-to-day tasks, including:</p>

<ol>
  <li>Collection-based fieldwork in fun places, such as the Southeast Asian
tropics and Gobi Desert.</li>
  <li>Molecular lab work to collect genetic data, which is becoming ever more
dominated by genomic library preparation for “next-generation” sequencing
(NGS) technology.</li>
  <li>Developing and implementing statistical methods that allow us to test our
hypotheses. This allows us to frame our work around questions that excite
us, rather than framing our questions around available methods.</li>
</ol>

<p>Below are summaries of some of our research.</p>

<hr />

<h2 id="inferring-shared-evolutionary-history">Inferring shared evolutionary history</h2>

<p>One of the major themes in our work is trying to better understand biological
diversification. To do so, it is important to account for processes that can
affect whole groups lineages.
In other words, if you imagine multiple lineages evolving through space and
time along the tree of life, there are a lot of potential processes that could
cause speciation and/or extinction to essentially co-occur across multiple
lineages.
If we are thinking on the scale of macrobiota, this can be any of the “usual
suspects” in biogeography (e.g., formation of geological features that cause
vicariance of whole communities of species).
If we are thinking on the scale of gene family evolution, this could be a
chromosomal duplication event.
If we want to learn about such processes from patterns of biological
diversity, we stand to benefit from explicitly incorporating them into
models of evolution.</p>

<p>Currently, phylogenetic models assume branching (speciation) events occur
independently (given the tree) across lineages. We are working on models that
relax this assumption, and allow clustering of branching events across
lineages. Our first attempt at relaxing this assumption is to infer the
distribution of divergence times across separate, rooted, two-taxon trees using
an approximate-likelihood Bayesian framework… I.e., starting simple!</p>

<h4 id="approximate-likelihood-bayesian-inference-of-co-diversification">Approximate-likelihood Bayesian inference of co-diversification</h4>

<p>The idea here is pretty simple. If, for example, we are interested in
whether or not three pairs of lineages diverged due to a common
mechanism, what we want to do is compare all five possible models
of their divergence (depicted in the cartoons below), which range
from complete dependence among their divergences (far left below)
to complete independence of their divergences (far right below).</p>

<figure>
    <a href="/images/three-lizard-div-models.png"><img src="/images/three-lizard-div-models.png" /></a>
    <figcaption>
        <p class="figure-caption-box">
            <span class="center-if-single-line">
                Cartoon examples of the five possible divergence models for
                three pairs of lineages.
            </span>
        </p>
    </figcaption>
</figure>

<p>The method implemented in the popular software package,
<a href="http://msbayes.sourceforge.net/">msBayes</a>, samples over these models, and uses
approximate-likelihoods and a basic Monte Carlo rejection algorithm to
approximate the posterior probability of each.
We extended this framework, developing a new method that uses flexible prior
probability distributions for many of the models’ parameters, and a
nonparametric Dirichlet-process prior across all possible divergence models
<a class="citation" href="#Oaks2014dpp">(Oaks, 2014)</a>.
We implemented the new model in the software package
<a href="https://github.com/joaks1/dpp-msbayes">dpp-msbayes</a>, and developed a much more
efficient multi-processing interface,
<a href="http://joaks1.github.io/PyMsBayes/">PyMsBayes</a>, to the models of both
<a href="https://github.com/joaks1/dpp-msbayes">dpp-msbayes</a> and
<a href="http://msbayes.sourceforge.net/">msBayes</a>.</p>

<h4 id="implementing-a-full-likelihood-bayesian-model-of-co-diversification">Implementing a full-likelihood Bayesian model of co-diversification</h4>

<p>While the method implemented in
<a href="https://github.com/joaks1/dpp-msbayes">dpp-msbayes</a> is more powerful and
robust than its predecessor, <a href="http://msbayes.sourceforge.net/">msBayes</a>, both
methods are based on approximate likelihoods and discard information when
reducing the genetic data to a set of insufficient summary statistics.
Thus, methods struggle to discriminate among models of divergence at
recent evolutionary timescales <a class="citation" href="#Oaks2012">(Oaks <span style="font-style: normal">et al.</span>, 2013; Oaks, Linkem &amp; Sukumaran, 2014; Oaks, 2014)</a>.
We are working on a method that employs a similar divergence-model-choice
approach to this problem, but in a full-likelihood Bayesian framework.
By utilizing all of the information in the data, we hope the method will be
much more accurate and precise than its approximate-Bayesian counterparts.
Ultimately, the goal is to incorporate this approach into a fully phylogenetic
framework, allowing us to relax the assumption of independence of branching
events across lineages and learn about shared processes of diversification via
phylogenetic inference.</p>

<hr />

<h2 id="diversification-across-southeast-asia">Diversification across Southeast Asia</h2>

<p>One question we are actively pursuing is whether Quaternary climatic
oscillations promoted diversification by fragmenting the distributions of
species.
An ideal model system for addressing this question is the dynamic landscape of
Southeast Asia.
Many groups of 26,000+ islands in this region coalesced during lower sea levels
of glacial periods and were fragmented during interglacial rises in sea level.</p>

<figure>
    <a href="/images/sunda-shelf.png"><img style="border:1px solid black" src="/images/sunda-shelf.png" /></a>
    <figcaption>
        <p class="figure-caption-box">
            <span class="center-if-single-line">
                Map of Southeast Asia with land extent (depicted in gray)
                during glacial lowstands estimated using 120m bathymetry (data
                from 
                <a href="http://ngdc.noaa.gov/mgg/global/global.html">ETOPO1</a>).
            </span>
        </p>
    </figcaption>
</figure>

<h4 id="diversification-of-geckos">Diversification of geckos</h4>

<p>To test whether the repeated formation and fragmentation of island complexes
promoted diversification, we are studying the evolutionary history of two
genera of geckos <em>Cyrtodactylus</em> and <em>Gekko</em> that are co-distributed across
most of the islands of the Philippines.
So far, we have found evidence of complex histories for both genera that
contradict many of the traditional areas of endemism predicted by island
connectivity during glacial periods <a class="citation" href="#Siler2010">(Siler <span style="font-style: normal">et al.</span>, 2010; Siler <span style="font-style: normal">et al.</span>, 2012)</a>.
Interestingly, for <em>Gekko</em>, we found evidence that the genus colonized the
Philippines by “rafting” on the Palawan Microcontinental Islands that rifted
away from Mainland Asia and drifted toward the rest of the Philippine
Archipelago <a class="citation" href="#Siler2012">(Siler <span style="font-style: normal">et al.</span>, 2012)</a>.</p>

<p>Currently, we are currently collecting genome-wide sequence data from nearly
300 individuals of <em>Cyrtodactylus</em> and <em>Gekko</em> from across the Philippine
Archipelago.
Combining these NGS data with new methods we are developing will allow us to
better assess the affect of sea-level fluctuations on diversification across
oceanic islands of Southeast Asia.</p>

<h4 id="comparative-phylogeography-of-the-philippines">Comparative phylogeography of the Philippines</h4>

<p>To more directly test whether glacial cycles promoted diversification, we are
also taking a broadly comparative approach.
For example, we are accumulating genetic data from island populations of
distantly related taxa distributed across the Philippine Islands
<a class="citation" href="#Oaks2012">(Oaks <span style="font-style: normal">et al.</span>, 2013)</a>.
If repeated bouts of island connectivity and isolation promoted
diversification, the temporal distribution of divergences across of
inter-island pairs of populations should be temporally clustered and correspond
to interglacial rises in sea level that fragmented the islands.
Our preliminary approach to this question, using single-locus data and
<a href="http://joaks1.github.io/PyMsBayes/">PyMsBayes</a>, suggested that multiple,
recent events were important in explaining the diversity across the sampled
taxa <a class="citation" href="#Oaks2014dpp">(Oaks, 2014)</a>.
However, due to a combination of limited data and our approximate-likelihood
approach, there was too much posterior uncertainty to be confident in any
divergence models.
To tackle this problem, we are working with collaborators to collect more
(genomic) data, and developing full-likelihood methods for inferring models of
divergence that can accommodate such data.</p>

<h4 id="diversification-across-mainland-southeast-asia">Diversification across mainland Southeast Asia</h4>

<p>We also want to better understand diversification in mainland Southeast Asia.
Along with collaborators, we are collecting large comparative datasets of
reptiles and amphibians from across Indochina and the Malay Peninsula.
We are excited about applying the methods we are developing to these data to
test long-standing hypotheses about the affect of biogeographical transition
zones on the diversification of Sunda-Shelf biota, including the Isthmus of
Kras, and current and paleo-river systems.</p>

<hr />

<h2 id="diversification-of-the-highly-endemic-fauna-of-the-gobi-desert">Diversification of the highly endemic fauna of the Gobi Desert</h2>

<p>The Gobi is the largest desert in Asia and is a mosaic of rocky mountain ranges
surrounded by basins of sand dunes.
As part of an international team of collaborators (Jesse Grismer and Rafe
Brown of the University of Kansas, Natalia Ananjeva of the Russian Academy
of Sciences, Xianguang Guo of the Chinese Institute of Biology, and
Nyamsuren Batsaikhan of the National University of Mongolia), we are working to
understand the evolutionary history of the unique assemblage of reptiles that
inhabit this region.
Our goal is to collect genomic data from a set of focal sand and rock-adapted
taxa and use the comparative methods we are developing to
(1) infer their evolutionary origins,
(2) test for patterns of shared evolutionary history, and
(3) identify key processes that led to the diverse and unique fauna of this
region.</p>

<hr />

<h2 id="diversification-in-west-central-african-rainforests">Diversification in West-Central African rainforests</h2>

<p>We are working with <a href="http://faculty.washington.edu/leache/wordpress/">Adam
Leaché</a> and <a href="http://www.uta.edu/faculty/mkfujita/">Matt
Fujita</a> to apply novel statistical
methods to comparative NGS datasets to elucidate the historical processes of
diversification and community assembly across West-Central African rainforests.
We are particularly interested in determining how Pliocene and Pleistocene
aridification cycles and associated rainforest fragmentation influenced
diversification across the Afro-tropics.</p>

<hr />

<h2 id="diversification-of-crocodiles">Diversification of Crocodiles</h2>

<div class="image-float-right-40">
    <a href="/images/croc-evol-cover.jpg"><img style="border:1px solid black" src="/images/croc-evol-cover.jpg" /></a>
    <br />
    <p class="figure-caption-box">
        <span class="center-if-single-line">
            <i>Crocodyuls siamensis</i>
        </span>
    </p>
</div>
<p>How and when did crocodiles get to where they are today? How many species are
there? Where and when did their common ancestor live?
Motivated by these questions, and the fact that crocodiles are awesome, I
collected a large dataset of DNA sequences from all extant crocodylian species
and used statistical phylogenetic methods to infer the temporal and
biogeographical origin of the “true” crocodiles, <em>Crocodylus</em>.
The leading hypothesis was that <em>Crocodylus</em> originated in Africa, however,
<em>when</em> was a lot more uncertain.
Traditionally, crocodiles were stereotyped as an ancient group of “living
fossils”, but molecular data and reassessment of paleontological data suggested
the genus might be younger than previously believed.</p>

<p>The DNA data I collected were strongly inconsistent with the hypotheses that
the common ancestor of <em>Crocodylus</em> was (1) ancient and (2) from Africa.
Rather, my results support that all extant crocodiles shared a common ancestor
from the Indo-Pacific, approximately 14–8 million years ago
<a class="citation" href="#Oaks2011">(Oaks, 2011)</a>.
This rejects a vicariant explanation of their biogeography in favor of a
recent, dispersal-mediated model involving multiple transoceanic dispersals.
Interestingly, it was not until after the mid-Miocene climatic optimum, when
the global climate was cooling and fellow crocodylian lineages were suffering
massive extinctions, that <em>Crocodylus</em> radiated and dispersed around the globe.
These findings raise interesting questions about how and why <em>Crocodylus</em> fared
so well during a period with the highest crocodilian extinction rate over the
last 100 million years.</p>

<figure>
    <a href="/images/croc-phylo.png"><img src="/images/croc-phylo-small.png" /></a>
    <figcaption>
        <p class="figure-caption-box">
            <span class="center-if-single-line">
                Summary of the fossil-calibrated species tree and
                biogeographical character-state reconstruction, supporting the
                <i>Crocodylus</i> ancestor within the Indo-Pacific and subsequent
                diversification after the mid-Miocene climatic optimum.
            </span>
        </p>
    </figcaption>
</figure>

<p>This work has been highlighted in:</p>

<ul>
  <li><a href="http://www.nature.com/nature/journal/v474/n7353/full/474545a.html">Nature</a></li>
  <li><a href="http://sciworthy.com/science-news/science-authors/whos-the-living-fossil/">Sciworthy</a></li>
  <li><a href="http://nautil.us/issue/22/slow/the-rise-and-fall-of-the-living-fossil">Nautilus</a></li>
</ul>

<hr />

<h2 id="data">Data</h2>

<ul>
  <li>We make our data available in repositories; please see our <a href="/publications/">papers</a> for
specific locations. Additionally, there are some “odds and ends” available
for download <a href="/research/data/">here</a>.</li>
</ul>

<hr />

<h2 id="references">References</h2>

<ol class="bibliography"><li><span id="Oaks2014reply">Oaks JR, Linkem CW, Sukumaran J. 2014. Implications of uniformly distributed, empirically informed priors for
phylogeographical model selection: A reply to Hickerson et al. <i>Evolution</i> <span style="font-weight: normal">68</span>:3607–3617. DOI: 10.1111/evo.12523.</span>

<a href="http://dx.doi.org/10.1111/evo.12523">Link.</a> 

</li>
<li><span id="Oaks2014dpp">Oaks JR. 2014. An Improved Approximate-Bayesian Model-choice Method for
        Estimating Shared Evolutionary History. <i>BMC Evolutionary Biology</i> <span style="font-weight: normal">14</span>:150. DOI: 10.1186/1471-2148-14-150.</span>

<a href="http://dx.doi.org/10.1186/1471-2148-14-150">Link.</a> 

</li>
<li><span id="Oaks2012">Oaks JR, Sukumaran J, Esselstyn JA, Linkem CW, Siler CD, Holder MT, Brown RM. 2013. Evidence for climate-driven diversification? A caution for interpreting ABC inferences of simultaneous historical events. <i>Evolution</i> <span style="font-weight: normal">67</span>:991–1010. DOI: 10.1111/j.1558-5646.2012.01840.x.</span>

<a href="http://dx.doi.org/10.1111/j.1558-5646.2012.01840.x">Link.</a> 

</li>
<li><span id="Siler2012">Siler CD, Oaks JR, Welton LJ, Linkem CW, Swab J, Diesmos AC, Brown RM. 2012. Did geckos ride the Palawan raft to the Philippines? <i>Journal of Biogeography</i> <span style="font-weight: normal">39</span>:1217–1234. DOI: 10.1111/j.1365-2699.2011.02680.x.</span>

<a href="http://dx.doi.org/10.1111/j.1365-2699.2011.02680.x">Link.</a> 

</li>
<li><span id="Oaks2011">Oaks JR. 2011. A time-calibrated species tree of Crocodylia reveals a recent radiation of the true crocodiles. <i>Evolution</i> <span style="font-weight: normal">65</span>:3285–3297. DOI: 10.1111/j.1558-5646.2011.01373.x.</span>

<a href="http://dx.doi.org/10.1111/j.1558-5646.2011.01373.x">Link.</a> 

</li>
<li><span id="Siler2010">Siler CD, Oaks JR, Esselstyn JA, Diesmos AC, Brown RM. 2010. Phylogeny and biogeography of Philippine bent-toed geckos (Gekkonidae: \emphCyrtodactylus) contradict a prevailing model of Pleistocene diversification. <i>Molecular Phylogenetics and Evolution</i> <span style="font-weight: normal">55</span>:699–710. DOI: 10.1016/j.ympev.2010.01.027.</span>

<a href="http://dx.doi.org/10.1016/j.ympev.2010.01.027">Link.</a> 

</li></ol>


      
        <hr />
        <div class="social-share">
  <h4>Share on</h4>
  <ul>
    <li>
    <a href="https://twitter.com/intent/tweet?text=http://phyletica.org/research/old-index/" class="twitter" title="Share on Twitter"><i class="fa fa-twitter"></i><span> Twitter</span></a>
    </li>
    <li>
      <a href="https://www.facebook.com/sharer/sharer.php?u=http://phyletica.org/research/old-index/" class="facebook" title="Share on Facebook"><i class="fa fa-facebook"></i><span> Facebook</span></a>
    </li>
    <li>
      <a href="https://plus.google.com/share?url=http://phyletica.org/research/old-index/" class="google-plus" title="Share on Google Plus"><i class="fa fa-google-plus"></i><span> Google+</span></a>
    </li>
  </ul>
</div><!-- /.social-share -->

      
    </div><!-- /.article-wrap -->
    <div class="article-author-bottom">
      

<div itemscope itemtype="http://schema.org/Person">


	<img src="/images/phyletica-avatar.png" class="bio-photo" alt="Phyletica bio photo">


  <h3 itemprop="name">Phyletica</h3>
  <p>Exploring biodiversification.</p>
  <a href="mailto:phyletica@auburn.edu" class="author-social" target="_blank"><i class="fa fa-fw fa-envelope-square"></i> Email</a>
  <a href="http://twitter.com/jamoaks" class="author-social" target="_blank"><i class="fa fa-fw fa-twitter-square"></i> Twitter</a>
  
  
  
  
  
  <a href="http://github.com/phyletica" class="author-social" target="_blank"><i class="fa fa-fw fa-github"></i> Github</a>
  
  
  
  
  
  
  <a href="https://youtube.com/channel/UCPPWy11zYqq-9lE_CQ0qPrw" class="author-social" target="_blank"><i class="fa fa-fw fa-youtube-square"></i> Youtube</a>
  
  
  
  <a href="http://phyletica.github.io" class="author-social" target="_blank"><i class="fa fa-fw fa-globe"></i> Interweb</a>
  <a href="http://www.auburn.edu" class="author-social" target="_blank"><i class="fa fa-fw fa-paw"></i> Auburn</a>
  <a href="http://www.auburn.edu/biology" class="author-social" target="_blank"><i class="fa fa-fw fa-university"></i> AU Biology</a>
  <a href="http://www.aumnh.org" class="author-social" target="_blank"><i class="fa fa-fw fa-leaf"></i> AU Museum</a>
  
  
</div>

    </div>
    
  </article>
</div><!-- /#index -->

<footer>
<div class="footer-wrap">
        
        
        <div class="center-footer-images">
            <a rel="au-phylo" href="http://www.auburn.edu/cosam/departments/biology/"><img class="footer-image" alt="Auburn Phylo" src="/images/auburn-phylo-mtan-2014-blue-256.png" /></a>
            <a rel="aumnh" href="http://aumnh.org/"><img class="footer-image" alt="Auburn MNH" src="/images/aumnh-crop-thumb.png" /></a>
        </div>
        <br>
        <p class="center-box">
            <span>
                Auburn-phylo graphic <a rel="mtan" href="https://sites.google.com/site/mtyourmind/">&copy; 2014 Milton Tan</a>.
            </span>
            <br />
            <span>
                Powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a>, based on the <a href="http://mademistakes.com/minimal-mistakes/" rel="nofollow">Minimal Mistakes</a> theme.
            </span>
            <br />
            <span>
                <a href="https://github.com/phyletica/phyletica.github.io" title="phyletica source on GitHub">View on GitHub</a>.
            </span>
            <br />
            <br />
            <span>
                <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>
                <br />
                Unless noted, content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
            </span>
        </p>
</div><!-- /.footer-wrap -->
</footer>


<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<script>window.jQuery || document.write('<script src="/assets/js/vendor/jquery-1.9.1.min.js"><\/script>')</script>
<script src="/assets/js/scripts.min.js"></script>
<script src="/assets/js/gif.js"></script>
<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<!-- Asynchronous Google Analytics snippet -->
<script>
  var _gaq = _gaq || [];
  var pluginUrl =
 '//www.google-analytics.com/plugins/ga/inpage_linkid.js';
  _gaq.push(['_require', 'inpage_linkid', pluginUrl]);
  _gaq.push(['_setAccount', 'UA-66934027-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>



</body>
</html>
